src/dev/failed_tests/report.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import xml2js from 'xml2js';
+import vfs from 'vinyl-fs';
+import es from 'event-stream';
+import { getGithubClient, markdownMetadata, paginate } from '../github_utils';
+import { find } from 'lodash';
+import stripAnsi from 'strip-ansi';
+
+const GITHUB_FLAKY_TEST_LABEL = 'flaky-failing-test';
```

Comment:
Flaky is kind of a misnomer here - it's really just a failing test. Can we change to `test-failure` or `failed-test`